cos_registration_server/applications/fields.py

```diff
@@ -0,0 +1,52 @@
+"""Custom YAML field."""
```

Comment:
Maybe let's keep a todo here regarding the license, since we took that from a repo.

Reply:
Added in 088a0785869d2e23d8ddfdf7f31589f8bc4327f4